supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-single-sign-on.adoc

```diff
@@ -0,0 +1,299 @@
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[[composite-role]]
+==== image:images/yes.png[yes] composite role
+*Description*: A composite role is a role that can be associated with other roles. For example a `superuser` composite role can be associated with the `sales-admin` and `order-entry-admin` roles. If a user is mapped to the `superuser` role they also inherit the `sales-admin` and `order-entry-admin` roles.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[consent]]
+==== image:images/yes.png[yes] consent
+*Description*: Consent is when you as an `admin` want a user to give permission to a client before that client can participate in the authentication process. After a user provides their credentials, Red Hat Single Sign-On opens a screen identifying the client requesting a login and what identity information is requested of the user. Users can decide whether or not to grant the request.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[credentials]]
+==== image:images/yes.png[yes] credentials
+*Description*: Credentials are pieces of data that Red Hat Single Sign-On uses to verify the identity of a user. Some examples are passwords, one-time passwords, digital certificates, or even fingerprints.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[direct-grant]]
+==== image:images/yes.png[yes] direct grant
+*Description*: A direct grant is a way for a client to obtain an access token on behalf of a user through a REST invocation.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[event-rhsso]]
+==== image:images/yes.png[yes] event
+*Description*: An event is an audit stream that administrators view and connect to.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[group]]
+==== image:images/yes.png[yes] group
+*Description*: A group manages a collection of users. You can define attributes for a group. You can also map roles to a group. Users that become members of a group inherit the attributes and role mappings that group defines.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[identity-provider]]
+==== image:images/yes.png[yes] identity provider
+*Description*: An identity provider (IDP) is a service that authenticates a user. Red Hat Single Sign-On is an IDP.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[identity-provider-federation]]
+==== image:images/yes.png[yes] identity provider federation
+*Description*: Red Hat Single Sign-On can be configured to delegate authentication to one or more IDPs. Social login through Facebook or Google+ is an example of identity provider federation. You can also hook Red Hat Single Sign-On to delegate authentication to any other OpenID Connect or SAML 2.0 IDP.
```

Comment:
```suggestion
*Description*: Red Hat Single Sign-On can be configured to delegate authentication to one or more IDPs. Social login through Facebook or Twitter is an example of identity provider federation. You can also connect Red Hat Single Sign-On to delegate authentication to any other OpenID Connect or SAML 2.0 IDP.
```

Reply:
Done @andymunro, thank you. Also suggesting to replace 'hook' with 'connect'  wdyt?

Reply:
Sounds a lot more professional.